internal/pkg/process/analyze.go

```diff
@@ -80,15 +82,24 @@ func (a *Analyzer) Analyze(pid int, relevantFuncs map[string]interface{}) (*Targ
 		return nil, err
 	}
 
-	goVersion, modules, err := a.getModuleDetails(elfF)
+	buildInfo, err := buildinfo.Read(f)
+	if err != nil {
+		return nil, err
+	}
+	goVersion, err := version.NewVersion(strings.ReplaceAll(buildInfo.GoVersion, "go", ""))
 	if err != nil {
 		return nil, err
 	}
 	result.GoVersion = goVersion
-
-	// Include the Go standard library module.
-	modules["std"] = goVersion
-	result.Libraries = modules
+	result.Libraries = make(map[string]*version.Version)
+	for _, dep := range buildInfo.Deps {
+		depVersion, err := version.NewVersion(dep.Version)
+		if err != nil {
+			return nil, err
```

Comment:
Not sure if we should return an error here. The current behavior is to log the error and continue. If I remember correctly, when parsing version with commit hashes in them, the parsing might fail.

Reply:
done